.gitignore

```diff
@@ -2,6 +2,7 @@
 build
 node_modules
 config/local.json
+packages/devtools-config/configs/local.json
```

Comment:
This doesn't need to be there. Feel free to `rm packages/devtools-config/configs/local.json` this.